src/main/java/Duke.java

```diff
@@ -1,10 +1,98 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+    public static final int MAX_SIZE = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner mySentence = new Scanner(System.in);
+        Task[] t = new Task[MAX_SIZE];
+        int ctr = 0;
+        welcomeMessage();
+        String sentence = "";
+        while (!Objects.equals(sentence, "bye")) {
+            sentence = mySentence.nextLine();
+            if (Objects.equals(sentence, "list")) {
+                printList(t, ctr);
+            } else if (sentence.contains("done")) {
+                markDone(t, sentence);
+            } else if (Objects.equals(sentence, "bye")) {
+                exitMessage();
+            } else {
+                ctr = createTask(t, ctr, sentence);
```

Comment:
Just a suggestion, I feel like createTask can be refactored and renamed createATask